front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -356,3 +355,28 @@ function StartHelperConversationButton({
     />
   );
 }
+interface Agent {
```

Comment:
This is a bit heavy to define an interface for such a simple component. Let's just have

```
const AvatarListItem = function({ agent } : { agent : { ... }}) { ... } 
```